test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -303,6 +305,85 @@ var _ = Describe("Mixedcpus", Ordered, func() {
 
 			})
 		})
+		When("Disabling mixedCpus in the performance profile", func() {
+			It("should causes all pods that uses shared CPUs to fail", func() {
+
+				By("Creating a deployment with one pod asking for a shared cpu")
+				rl := &corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("1"),
+					corev1.ResourceMemory: resource.MustParse("100Mi"),
+					sharedCpusResource:    resource.MustParse("1"),
+				}
+				p := makePod(ctx, testclient.Client, testutils.NamespaceTesting,
+					withRequests(rl),
+					withLimits(rl),
+					withRuntime(components.GetComponentName(profile.Name, components.ComponentNamePrefix)))
+				dp := deployments.Make(testutils.DeploymentTesting, testutils.NamespaceTesting,
+					deployments.WithPodTemplate(p),
+					deployments.WithNodeSelector(testutils.NodeSelectorLabels))
+
+				Expect(testclient.Client.Create(ctx, dp)).ToNot(HaveOccurred())
+				podList := &corev1.PodList{}
+				listOptions := &client.ListOptions{Namespace: testutils.NamespaceTesting, LabelSelector: labels.SelectorFromSet(dp.Spec.Selector.MatchLabels)}
+				Eventually(func() bool {
+					isReady, err := deployments.IsReady(ctx, testclient.Client, listOptions, podList, dp)
+					Expect(err).ToNot(HaveOccurred())
+					return isReady
+				}, time.Minute, time.Second).Should(BeTrue())
+				Expect(testclient.Client.List(ctx, podList, listOptions)).To(Succeed())
```

Comment:
Is `To(Succeed)` is the same as `ToNot(HaveOccured)`?

Reply:
```
func Succeed() types.GomegaMatcher
Succeed passes if actual is a nil error Succeed is intended to be used with functions that return a single error value. Instead of

err := SomethingThatMightFail()
Expect(err).ShouldNot(HaveOccurred())
You can write:

Expect(SomethingThatMightFail()).Should(Succeed())
```
According to the docs, in this case it should have the same effect.